I_dont_have_vinaigrette.py

```diff
@@ -0,0 +1,42 @@
+import random
+import time
+import datetime
+
+
+# This program generates random date between two dates(start,end)
+# If the date is of a Monday (Monday is 0 in weekday() return value
+# The program will print "I don't have Vinaigrette"
+# (Which I find a little wierd, I mean, who even likes vinaigrette that much?)
+
+def str_time_prop(start, end, time_format, prop):
```

Comment:
rewrite this function to look like this
![Capture](https://user-images.githubusercontent.com/13467749/160186339-81fb3d14-9342-4bd9-8a0b-62d3e7d56855.JPG)